src/adapters/__tests__/url.test.js

```diff
@@ -2,10 +2,17 @@ import urlAdapter from '../url'
 
 const getParams = () =>
   new URLSearchParams(new URL(window.location.href).search)
+const getHash = () =>
+  window.location.hash
 const persist = (params = '') =>
-  window.history.pushState({}, '', `${window.location.pathname}?${params}`)
+  setActiveURL(`${window.location.pathname}?${params}`)
+const resetURL = () =>
+  setActiveURL('/')
+const setActiveURL =  (url) =>
```

Comment:
Why not just reuse persist and pass it a second hash argument instead?

Reply:
That would mean that either `${window.location.pathname}?${params}` would be the default value for this argument, or I would have to add it to all the tests that call `persist`. Either way is fine by me, but I felt that this would be a bit cleaner. What do you think?

Reply:
It seems cleaner to me to have a single function which takes two arguments, both with a default empty value. This avoid having too many functions and keeps the test easy to read.

```js
const persist = (params = '', hash = '') => window.history.pushState({}, '', `${window.location.pathname}?${params}${hash}`)
```